contracts/contracts/UserReplicaSetManager.sol

```diff
@@ -146,51 +161,81 @@ contract UserReplicaSetManager is SigningLogicInitializable, RegistryContract {
         // For every entry in spIds/Nonces/Sigs
         //  Recover signer using the signature for inner function (tmp is addOrUpdateCreatorNode)
         //  Confirm that the spId <-> recoveredSigner DOES exist and match what is stored on chain
-        address proposer1Address = _recoverProposeAddOrUpdateContentNodeSignerAddress(
+        address proposer1DelegateOwnerWallet = _recoverProposeAddOrUpdateContentNodeSignerAddress(
             _cnodeSpId,
-            _cnodeDelegateOwnerWallet,
+            _cnodeWallets[0],
+            _cnodeWallets[1],
             _proposerSpIds[0],
             _proposerNonces[0],
             _proposer1Sig
         );
-        address proposer2Address = _recoverProposeAddOrUpdateContentNodeSignerAddress(
+        address proposer2DelegateOwnerWallet = _recoverProposeAddOrUpdateContentNodeSignerAddress(
             _cnodeSpId,
-            _cnodeDelegateOwnerWallet,
+            _cnodeWallets[0],
+            _cnodeWallets[1],
             _proposerSpIds[1],
             _proposerNonces[1],
             _proposer2Sig
         );
-        address proposer3Address = _recoverProposeAddOrUpdateContentNodeSignerAddress(
+        address proposer3DelegateOwnerWallet = _recoverProposeAddOrUpdateContentNodeSignerAddress(
             _cnodeSpId,
-            _cnodeDelegateOwnerWallet,
+            _cnodeWallets[0],
+            _cnodeWallets[1],
             _proposerSpIds[2],
             _proposerNonces[2],
             _proposer3Sig
         );
-        _validateUpdateOperation(
-            proposer1Address,
-            proposer2Address,
-            proposer3Address,
-            _proposerSpIds
+        require(
+            (proposer1DelegateOwnerWallet != proposer2DelegateOwnerWallet) &&
+            (proposer1DelegateOwnerWallet != proposer3DelegateOwnerWallet) &&
+            (proposer2DelegateOwnerWallet != proposer3DelegateOwnerWallet),
+            "Distinct proposer delegateOwnerWallets required"
+        );
+        require(
+            spIdToContentNodeWallets[_proposerSpIds[0]].delegateOwnerWallet == proposer1DelegateOwnerWallet,
+            "Invalid wallet provided for 1st proposer"
```

Comment:
lets clarify here that it is "Invalid delegateOwnerWallet"?

Reply:
sure